src/main/kotlin/racingcar/domain/CarNameWithLocation.kt

```diff
@@ -0,0 +1,8 @@
+package racingcar.domain
+
+data class CarNameWithLocation(
+    private val value: Pair<CarName, Location>
+) {
```

Comment:
```suggestion
data class CarNameWithLocation(
    val carName: CarName,
    val location: Location
) {
```
제가 의도한 것은 Pair를 data class로 wrapping 하는 것이 아니라, 요소들을 풀어서 wrapping 하는 것을 의도했어요